### ISSUE TITLE
Yelp Dataset Question

### ISSUE THREAD

**Issue by JingyuanHe1222** [OpenMatch/TASTE#12]
Hi, thanks for the nice work!

I am trying to reproduce your work and is checking the datasets from DIF-SR, which is cited as the source data in your source code. 
It seems like DIF-SR provides multiple versions of Yelp. Are you using the latest one? The Yelp2022 set? 

Thanks in advance!

**Comment by mssssss123** [comment]
Hello,
I checked the RecBole dataset website and it seems they updated the dataset causing your confusion. I compared their data that I saved in the past, and it should be the version selected in the picture, yelp2020. (The link is as follows: https://drive.google.com/drive/folders/1n_wODEhexezzI1yJyQP_PEBDrKNV0mEG).
At the same time, Amazon has also updated, it should be this link (https://drive.google.com/drive/folders/1FUAScpZtCmArqQS0xdrIqpFx5F7u3TaY).
If you still have problems, I can share with you my past version and hope it can solve your problem.
<img width="402" alt="截屏2024-05-24 21 50 49" src="https://github.com/OpenMatch/TASTE/assets/56372416/4b4d9096-1e16-4f7e-aaa7-a63a7dc38dce">
